.gitignore

```diff
@@ -5,3 +5,4 @@ examples/terraform.tfstate
 **/*/*.tfstate.backup
 dist/
 terraform-provider-firehydrant
+.idea
```

Comment:
probably belong to your personal global ignore not the project :D

Reply:
😄 Whoops

Reply:
it happens :D